hadoop-ozone/tools/pom.xml

```
@@ -124,6 +124,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+    </dependency>
```

Should this have a scope of `test`?

```suggestion
      <artifactId>junit-jupiter-params</artifactId>
      <scope>test</scope>
    </dependency>
```

---

yes, IDE auto imported it and I forgot to check. I'll fix it!